src/main/groovy/org/boozallen/plugins/jte/job/TemplateFlowDefinition.groovy

```diff
@@ -52,6 +53,14 @@ abstract class TemplateFlowDefinition extends FlowDefinition {
     FlowExecution create(FlowExecutionOwner owner, TaskListener listener, List<? extends Action> actions) throws Exception {
         FlowDurabilityHint hint = determineFlowDurabilityHint(owner)
         String template = initializeJTE(owner)
+
```

Comment:
The action usage should happen before the initializeJTE call. basically put the action loop at the top

Reply:
I tried that, but then it failed to find any of the JTE variables when it ran. The hint part can probably move after the actions, but the initializeJTE part needs to happen first.

Reply:
ok. let me get some feedback from Steven